### ISSUE TITLE
[Due for payment 2026-04-03] [$250] Screen Reader: Settings-Profile: No information of context menu and how to open it

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76948]
## Action Performed:
Prerequisite:
User is logged in
1. Using Windows + Chrome + JAWS, open the page https://new.expensify.com/settings/profile
2. Navigate using TAB to the 'Help' and 'What's New' links in the 'General' section
3. Observe there is no announcement for the context menus available

Other Occurrences:
"1. On Settings - About - App Download Links: the links of downloads
2. On Chat: the menu of contacts"

## Expected Result:
When users navigate through the profile page, onto the general section, users will be informed there is a context menu with information for the 'Help' and 'What's New' links.
-"Help, link, Context menu available, press SHIFT F10 to activate"

## Actual Result:
When users navigate to the 'Help' and 'What's new' links in the account menu, there is no indication that an additional context menu is available. However, if the user right clicks on either of the links, a 'Copy to Clipboard' menu opens, which is not announced at all to the user. Additionally, the user can press SHIFT + F10 to reach the menu, but that information is not conveyed to the user.

HTML:
<div aria-label="Copy URL to clipboard" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-17gur6a r-18u37iz r-1wtj0ep r-1mmae3n r-u9wvl5 r-13qz1uu r-1ksorx6 r-1jsugan r-85oauj r-17lp2du" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style=""><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci" style="width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="true" pressed="false" fill="#03D47C"><path d="M8 0C6.9 0 6 .9 6 2v12c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V4.8c0-.5-.2-1-.6-1.4L14.6.6c-.4-.4-.9-.6-1.4-.6z"></path><path d="M2 6c0-1.1.9-2 2-2v12c0 1.1.9 2 2 2h8c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2z"></path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1 r-1udh08x r-1udbk01 r-1iln25a" style="color: rgb(0, 46, 34); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; flex-shrink: 1; margin-left: 12px; white-space: pre; direction: ltr; max-width: 100%; line-height: 20px;">Copy URL to clipboard</div></div><div dir="auto" class="css-146c3p1 r-8akbws r-krxsd3 r-dnmrzs r-1qsk4np r-1udbk01" style="-webkit-line-clamp: 2; color: rgb(118, 132, 126); font-size: 13px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; margin-left: 12px; margin-top: 4px; line-height: 16px; word-break: break-word;">[https://help.expensify.com</div></div></div></div><div](https://help.expensify.com%3C/div%3E%3C/div%3E%3C/div%3E%3C/div%3E%3Cdiv) class="css-g5y9jx r-18u37iz" style="min-height: 40px;"></div></div></div></div>

## Area issue was found in:
Settings–Profile

## Failed WCAG checkpoints
WCAG 2.1.1 Keyboard
https://www.w3.org/WAI/WCAG22/Understanding/keyboard

## User impact:
People who are blind cannot use pointing devices like a mouse that require eye-hand coordination. People with low vision may have trouble finding or tracking a pointer indicator on screen. Some people with hand tremors find using a mouse very difficult and therefore usually use a keyboard.

## Suggested resolution:
It is recommended to provide a generic help somewhere to inform all users how to open the context menus and where they are available.

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1920" height="1004" alt="Image" src="https://github.com/user-attachments/assets/9fd3f102-a621-4329-a95c-85e5269facc1" />

https://github.com/user-attachments/assets/186d1130-b4ad-4a12-bf81-9c9d41d06b35

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6999608
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022032306195981069205</li>
        <li>Upwork Job ID: 2032306195981069205</li>
        <li>Last Price Increase: 2026-03-13</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
🤖 Draft PR created: https://github.com/Expensify/App/pull/85174
Please mark it as "Ready for review" when it is ready for review.

**Comment by marufsharifi** [comment]
@mallenexpensify, I've accepted the offer. thanks.

**Comment by truph01** [comment]
# BugZero Checklist:

- [ ] **[Contributor]** The offending PR and associated issue have been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake.

    Link to the comment on the PR:
    Link to the comment on the Issue:

- [x] **[Contributor]** If the regression was CRITICAL (e.g. interrupts a core flow) A discussion in [#expensify-open-source](https://app.slack.com/client/E047TPA624F/C01GTK53T8Q) has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner.

    Link to discussion:

- [x] **[Contributor]** If it was decided to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps using the template below to ensure the same bug will not reach production again.


- [x] **[BugZero Assignee]** Create a GH issue for creating/updating the regression test once above steps have been agreed upon.

    Link to issue:

## Regression Test Proposal
### Precondition:




### Test:


1. Using Windows + Chrome + JAWS, open the page /settings/profile
2. Navigate using TAB to the 'Help' and 'What's New' links in the 'General' section
3. Observe that there is no announcement for the context menus available
4. When users navigate through the profile page, onto the general section, verify that it informs that there is a context menu with information for the 'Help' and 'What's New' links.
-"Help, link, Context menu available, press SHIFT F10 to activate."

Do we agree 👍 or 👎

**Comment by garrettmknight** [comment]
$250 approved for @truph01
